app.mjs

```diff
@@ -22,9 +22,12 @@ import cors from 'cors'
 import indexRouter from './index.mjs'
 import manifestRouter from './manifest/index.mjs'
 import projectRouter from './project/index.mjs'
+import projectsRouter from './project/projects.mjs'
```

Comment:
our pattern everywhere else is for `index.mjs` in these. I think we should stick with it.

Reply:
in some cases, both exist and are redundant with each other